src/main/java/com/nike/cerberus/domain/environment/Stack.java

```diff
@@ -32,18 +32,17 @@
  */
 public class Stack implements Comparable<Stack> {
 
-    public static final Stack BASE = new Stack("base", "base.yaml", false);
-    public static final Stack VAULT = new Stack("vault", null, false);
+    public static final Stack CMS_IAM_ROLE = new Stack("cms-iam-role", "cms-iam-role.yaml", false);
+    public static final Stack REGION_BASE = new Stack("region-base", "region-base.yaml", false);
```

Comment:
nit: Why "region-base"? The added "region" seems to only make it more confusing.

I think just "base" is fine

Reply:
Your commenting on outdated files